__tests__/app.test.js

```diff
@@ -199,3 +198,40 @@ describe("PATCH /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/reviews", () => {
+  test("200: responds with a reviews array", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        console.log(reviews);
+        expect(Array.isArray(reviews)).toBe(true);
+        expect(reviews.length).toBe(13);
+        reviews.forEach((review) => {
+          expect(review).toMatchObject({
+            review_id: expect.any(Number),
+            title: expect.any(String),
+            designer: expect.any(String),
+            owner: expect.any(String),
+            review_img_url: expect.any(String),
+            review_body: expect.any(String),
+            category: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("404: responds when endpoint doesnt exist and/or is spelt wrong", () => {
```

Comment:
you should have hopefully added this kind of test already in your default 404 handler as such this test doesnt need to be added here